Content.Server/_Goobstation/Devil/DevilSystem.Handshake.cs

```diff
@@ -0,0 +1,120 @@
+// SPDX-FileCopyrightText: 2025 GoobBot <uristmchands@proton.me>
+// SPDX-FileCopyrightText: 2025 Solstice <solsticeofthewinter@gmail.com>
+// SPDX-FileCopyrightText: 2025 SolsticeOfTheWinter <solsticeofthewinter@gmail.com>
+// SPDX-FileCopyrightText: 2025 gus <august.eymann@gmail.com>
+//
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+using Content.Shared._Goobstation.CheatDeath;
+using Content.Shared._Goobstation.Devil;
+using Content.Shared._Goobstation.Devil.Condemned;
+using Content.Shared.Body.Part;
+using Content.Shared.Mobs.Components;
+using Content.Shared.Popups;
+using Content.Shared.Verbs;
```

Comment:
unused using directive
```suggestion
using Content.Shared.Mobs.Components;
using Content.Shared.Verbs;
```